routes/index.js

```diff
@@ -1,7 +1,13 @@
-import product from './product';
+import signUp from './auth/signUp';
+import signIn from './auth/signIn';
+import facebook from './auth/facebook';
 import users from './users';
+import product from './product';
 
 export default (app) => {
-	app.use('/api/products', product);
+	app.use('/api/signup', signUp);
+	app.use('/api/signin', signIn);
 	app.use('/api/users', users);
```

Comment:
The `isAuth` middleware should be applied here, therefore this middleware will be applicable only for the the routes not bound to authentication but to Products and Users access

Reply:
And also remove the apply of this middleware in other files

Reply:
Just re-read the theory of Express middlewares